site/docs/status.md

```diff
@@ -0,0 +1,362 @@
+|----------------------------|------|-----------|------|----|
+| Plan with data file        | Y    | N         | Y    | N  |
+| Plan with position deletes | Y    | N         | N    | N  |
+| Plan with equality deletes | Y    | N         | N    | N  |
+| Read data file             | Y    | N         | Y    | N  |
+| Read with position deletes | Y    | N         | N    | N  |
+| Read with equality deletes | Y    | N         | N    | N  |
+
+## Table Write Operations
+
+### Table Spec V1
+
+| Operation   | Java | PyIceberg | Rust | Go |
+|-------------|------|-----------|------|----|
+| Append data | Y    | N         | Y    | N  |
+
+### Table Spec V2
+
+| Operation              | Java | PyIceberg | Rust | Go |
+|------------------------|------|-----------|------|----|
+| Append data            | Y    | N         | Y    | N  |
+| Write position deletes | Y    | N         | N    | N  |
+| Write equality deletes | Y    | N         | N    | N  |
+
+## Catalogs
+
+### Rest Catalog
+
+#### Table Spec V1
+
+| Table Operation | Java | PyIceberg | Rust | Go |
+|-----------------|------|-----------|------|----|
+| listTable       | Y    | Y         | Y    | Y  |
+| createTable     | Y    | Y         | Y    | Y  |
+| dropTable       | Y    | Y         | Y    | Y  |
+| loadTable       | Y    | Y         | Y    | Y  |
+| updateTable     | Y    | Y         | Y    | Y  |
+| renameTable     | Y    | Y         | Y    | Y  |
+| tableExists     | Y    | Y         | Y    | Y  |
+
+#### Table Spec V2
+
+| Table Operation | Java | PyIceberg | Rust | Go |
+|-----------------|------|-----------|------|----|
+| listTable       | Y    | Y         | Y    | Y  |
+| createTable     | Y    | Y         | Y    | Y  |
+| dropTable       | Y    | Y         | Y    | Y  |
+| loadTable       | Y    | Y         | Y    | Y  |
+| updateTable     | Y    | Y         | Y    | Y  |
+| renameTable     | Y    | Y         | Y    | Y  |
+| tableExists     | Y    | Y         | Y    | Y  |
+
+#### View Spec V1
+
+| View Operation | Java | PyIceberg | Rust | Go |
+|----------------|------|-----------|------|----|
+| createView     | Y    | N         | N    | N  |
+| dropView       | Y    | N         | N    | N  |
+| listView       | Y    | N         | N    | N  |
+| viewExists     | Y    | N         | N    | N  |
+| replaceView    | Y    | N         | N    | N  |
+| renameView     | Y    | N         | N    | N  |
+
+#### Namespace Operations
+
+| Namespace Operation | Java | PyIceberg | Rust | Go |
+|---------------------|------|-----------|------|----|
+| listNamespaces      | Y    | N         | N    | N  |
+| createNamespace     | Y    | N         | N    | N  |
+| dropNamespace       | Y    | N         | N    | N  |
+| namespaceExists     | Y    | N         | N    | N  |
+| renameNamespace     | Y    | N         | N    | N  |
+| updateNamespace     | Y    | N         | N    | N  |
+| deleteNamespace     | Y    | N         | N    | N  |
+| loadNamespace       | Y    | N         | N    | N  |
```

Comment:
I think `deleteNamespace` and `dropNamespace` may be the same operation. And `loadNamespace` operation may be `listTables` instead?

Updated with PyIceberg supported features below

```suggestion
| listNamespaces                | Y    | Y         | N    | N  |
| createNamespace               | Y    | Y         | N    | N  |
| dropNamespace                 | Y    | Y         | N    | N  |
| namespaceExists               | Y    | N         | N    | N  |
| renameNamespace               | Y    | N         | N    | N  |
| updateNamespaceProperties     | Y    | Y         | N    | N  |
```